src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -1621,22 +1621,6 @@ bool CanReuse(ProjectId id)
             }
         }
 
-        /// <summary>
-        /// Gets a copy of the solution isolated from the original so that they do not share computed state.
-        ///
-        /// Use isolated solutions when doing operations that are likely to access a lot of text,
-        /// syntax trees or compilations that are unlikely to be needed again after the operation is done.
-        /// When the isolated solution is reclaimed so will the computed state.
-        /// </summary>
-        public SolutionState GetIsolatedSolution()
-        {
-            var forkedMap = ImmutableDictionary.CreateRange(
-                _projectIdToTrackerMap.Where(kvp => kvp.Value.HasCompilation)
```

Comment:
how important is HasCompilation now?  can that be removed in teh future?

Reply:
It's used in CreateCompilationTrackerMap when forking the Solution. I admit the use is supicious looking, but it's not something I'd want to fix here.